storybook/qmlTests/tests/tst_ManageCollectiblesPanel.qml

```diff
@@ -0,0 +1,308 @@
+            verify(!!switchArrangeByCommunity)
+            switchArrangeByCommunity.toggle()
+            const lvCommunityTokenGroups = findChild(loaderCommunityTokens, "lvCommunityTokenGroups")
+            verify(!!lvCommunityTokenGroups)
+
+            // verify we have 2 community collectible groups
+            tryCompare(lvCommunityTokenGroups, "count", 2)
+            triggerDelegateMenuAction(lvCommunityTokenGroups, 0, "miHideTokenGroup", true)
+
+            verify(controlUnderTest.dirty)
+
+            // verify we have one less group
+            waitForItemPolished(lvCommunityTokenGroups)
+            tryCompare(lvCommunityTokenGroups, "count", 1)
+            const lvHidden = findChild(controlUnderTest, "lvHiddenTokens")
+            verify(!!lvHidden)
+            tryCompare(lvHidden, "count", 4) // we've just hidden 4 collectibles coming from this group
+
+            verify(controlUnderTest.dirty)
+
+            // verify hidden items are not draggable
+            const hiddenToken = findChild(lvHidden, "manageTokensDelegate-0")
+            verify(!!hiddenToken)
+            compare(hiddenToken.dragEnabled, false)
+            const hiddenDraggable = findChild(hiddenToken, "draggableDelegate")
+            verify(!!hiddenDraggable)
+            mousePress(hiddenToken)
+            tryCompare(hiddenDraggable, "dragActive", false)
+            mouseRelease(hiddenToken)
+
+            // now show one of the 4 hidden tokens
+            waitForItemPolished(lvHidden)
+            triggerDelegateMenuAction(lvHidden, 0, "miShowToken")
+
+            verify(controlUnderTest.dirty)
+
+            // verify we again have 2 community groups, and one less hidden token
+            tryCompare(lvCommunityTokenGroups, "count", 2)
+            tryCompare(lvHidden, "count", 3)
+
+            verify(controlUnderTest.dirty)
+
+            // now mass show tokens from this group, verify we have 0 hidden tokens and 2 visible groups
+            triggerDelegateMenuAction(lvHidden, 0, "miShowTokenGroup")
+            waitForItemPolished(lvHidden)
+            tryCompare(lvHidden, "count", 0)
+            tryCompare(lvCommunityTokenGroups, "count", 2)
+
+            verify(controlUnderTest.dirty)
+        }
+
+        function test_dnd() {
+            verify(!controlUnderTest.dirty)
+
+            const lvRegular = findChild(controlUnderTest, "lvRegularTokens")
+            verify(!!lvRegular)
+            verify(lvRegular.count !== 0)
+
+            const delegate0 = findChild(lvRegular, "manageTokensDelegate-0")
+            verify(!!delegate0)
+            const title0 = delegate0.title
+            verify(!!title0)
+            const title1 = findChild(lvRegular, "manageTokensDelegate-1").title
+            verify(!!title1)
+
+            // DND one item down (~80px in height)
+            mouseDrag(delegate0, delegate0.width/2, delegate0.height/2, 0, 80)
```

Comment:
I wonder if maybe instead of using fixed value like `80` it could be possible to use value derived from delegate height, to make the test more resistant to pure-ui changes like changing delegate's height.

Reply:
Yeah, I was also thinking about that... but it's hard, the height can be variable

Reply:
But it's fine imo... if we ever change the delegate height for whatever reason, this test will start failing so we definitely notice

Reply:
> But it's fine imo... if we ever change the delegate height for whatever reason, this test will start failing so we definitely notice

Yea, we will notice but will need to fix. But if it's problematic, than ok, nothing important.